frontend/pages/hosts/details/cards/Software/SelfService/SelfServiceTableConfig.tsx

```diff
@@ -102,22 +104,81 @@ interface InstallUuid {
 
 export type InstallOrCommandUuid = CommandUuid | InstallUuid;
 
+export type ScriptExecutionId = {
+  script_execution_id: string;
+};
+
 type IInstallerStatusProps = Pick<IHostSoftware, "status"> & {
   last_install: ISoftwareLastInstall | IAppLastInstall | null;
+  last_uninstall: ISoftwareLastUninstall | IAppLastUninstall | null;
   onShowInstallerDetails: (uuid?: InstallOrCommandUuid) => void;
+  onShowUninstallDetails: (ScriptExecutionId?: ScriptExecutionId) => void;
 };
 
 const InstallerStatus = ({
   status,
   last_install,
+  last_uninstall,
   onShowInstallerDetails,
+  onShowUninstallDetails,
 }: IInstallerStatusProps) => {
   const displayConfig = STATUS_CONFIG[status as keyof typeof STATUS_CONFIG];
   if (!displayConfig) {
     // This is shown mid-install
     return <>{DEFAULT_EMPTY_CELL_VALUE}</>;
   }
 
+  // TODO: If there's both a last_install and last_uninstall, render the most recent status
```

Comment:
Unreleased bug I found while fixing bugs Janis found:

last_install and last_uninstall can both be present, IMO to dev this properly I could either 1) compare timestamps and render the most recent OR b) use status value to determine which of the two we render. Anyone have preference? 
- est: 1 point